esp-hal/src/uart.rs

```diff
@@ -48,12 +56,12 @@
 //! ### Sending and Receiving Data
 //! ```rust, no_run
 #![doc = crate::before_snippet!()]
-//! # use esp_hal::uart::{config::Config, Uart};
-//! use esp_hal::gpio::Io;
+//! # use esp_hal::uart::{self, Uart};
+//! # use esp_hal::gpio::Io;
 //! # let io = Io::new(peripherals.GPIO, peripherals.IO_MUX);
 //! # let mut uart1 = Uart::new_with_config(
 //! #     peripherals.UART1,
-//! #     Config::default(),
+//! #     uart::config::Config::default(),
```

Comment:
@MabezDev do you want me to undo changes like this (where it's visible), or can we keep them?

Reply:
Not a problem, just a lot to read, and a bunch of update-and-see-CI :)

Reply:
Narrator: 90 minutes later...